packs/data/age-of-ashes-bestiary.db/warbal-bumblebrasher.json

```diff
@@ -498,7 +499,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p><strong>Trigger</strong> A goblin ally ends a move action adjacent to Warbal.</p>\n<hr />\n<p><strong>Effect</strong> Warbal @Compendium[pf2e.actionspf2e.Step]{Steps}.</p>"
+                    "value": "<p><strong>Trigger</strong> A goblin ally ends a move action adjacent to the goblin.</p>\n<hr />\n<p><strong>Effect</strong> The goblin @Compendium[pf2e.actionspf2e.Step]{Steps}.</p>"
```

Comment:
Do we really want to change the text to not match the book?

Reply:
I've reverted the text. I also removed Goblin Scuttle from the Family Glossary, as it shouldn't have been added in the first place based on our policies.